answerkey/laziness/14.answer.scala

```diff
@@ -4,4 +4,10 @@
 def startsWith[A](s: Stream[A]): Boolean = 
   zipAll(s).takeWhile(!_._2.isEmpty) forAll {
     case (h,h2) => h == h2
-  }
\ No newline at end of file
+  }
+
+def startsWith[A](s: Stream[A]): Boolean = 
```

Comment:
For example, when `Stream(1, 2).startsWith(Stream(1))`, it should return true, but I feel this implementation would return false.